gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -1374,58 +1373,21 @@ protected void configureRestModelGeneration(Project project, SourceSet sourceSet
             task.setSuffix(IDL_FILE_SUFFIX);
 
             task.onlyIf(t ->
-            {
-              project.getLogger().info(
-                  "SKIP_IDL: {}\n" +
-                  "IDL_NO_PUBLISH: {}\n" +
-                  "SNAPSHOT_NO_PUBLISH: {}\n" +
-                  "checkRestModelTask:" +
-                  " Executed: {}" +
-                  ", Not Skipped: {}" +
-                  ", No Failure: {}" +
-                  ", Is Not Equivalent: {}\n" +
-                  "checkIdlTask:" +
-                  " Executed: {}" +
-                  ", Not Skipped: {}" +
-                  ", No Failure: {}" +
-                  ", Is Not Equivalent: {}\n" +
-                  "checkSnapshotTask:" +
-                  " Executed: {}" +
-                  ", Not Skipped: {}" +
-                  ", No Failure: {}" +
-                  ", Is RestSpec Not Equivalent: {}\n",
-                  isPropertyTrue(project, SKIP_IDL_CHECK),
-                  isPropertyTrue(project, IDL_NO_PUBLISH),
-                  isPropertyTrue(project, SNAPSHOT_NO_PUBLISH),
-                  checkRestModelTask.getState().getExecuted(),
-                  !checkRestModelTask.getState().getSkipped(),
-                  checkRestModelTask.getState().getFailure() == null,
-                  !checkRestModelTask.isEquivalent(),
-                  checkIdlTask.getState().getExecuted(),
-                  !checkIdlTask.getState().getSkipped(),
-                  checkIdlTask.getState().getFailure() == null,
-                  !checkIdlTask.isEquivalent(),
-                  checkSnapshotTask.getState().getExecuted(),
-                  !checkSnapshotTask.getState().getSkipped(),
-                  checkSnapshotTask.getState().getFailure() == null,
-                  !checkSnapshotTask.isEquivalent());
-
-              return !isPropertyTrue(project, IDL_NO_PUBLISH) &&
-              (
-                (
-                   isPropertyTrue(project, SKIP_IDL_CHECK) &&
-                   isTaskSuccessful(checkSnapshotTask) &&
-                   !(checkSnapshotTask.isRestSpecEquivalent())
-                ) ||
-                (
-                   !isPropertyTrue(project, SKIP_IDL_CHECK) &&
-                   (
-                      (isTaskSuccessful(checkRestModelTask) && !(checkRestModelTask.isRestSpecEquivalent())) ||
-                      (isTaskSuccessful(checkIdlTask) && !(checkIdlTask.isEquivalent()))
-                   )
-                )
-              );
-            });
+                    !isPropertyTrue(project, IDL_NO_PUBLISH) &&
```

Comment:
maybe at least logging whether the task is successful?
Personally I prefer to log all `executed`, `skipped` `uptodate` status.

Reply:
Gradle already logs all these details.